script.js

```diff
@@ -457,26 +466,18 @@ function handleCitiesLayer(geojson) {
 
   return citiesLayer;
 }
+//FIXME: city colors are working but the states and counties are not.
 
 function handleCountiesLayer(geojson) {
   const layerOptions = {
     style: feature => {
       // style counties based on whether their moratorium has passed
```

Comment:
@jimboweb please update or remove this comment `// style counties based on whether their moratorium has passed`